apps/docs/assets/styles/icons-gallery.css

```
@@ -0,0 +1,60 @@
+.div-wrapper{
+    max-width: 1200px;
+}
+
+#iconsTextField {
+    display: flex;
+    margin-top: 32px;
+}
+
+#iconsLayout {
+    --layout-column-gap: 16px; 
+    --layout-row-gap: 16px;
+    --layout-grid-template-columns: repeat(auto-fill, minmax(150px, 150px));
```

```suggestion
    --layout-grid-template-columns: repeat(auto-fill, minmax(150px, 1fr));
```it looks better in mobile